### PR TITLE
Update _index.md Added links and descriptions of tutorials

### PR DESCRIPTION
**Pull request by memartone**
I realized that the side bar labels don't really say what the tutorial is about so I think we need the list of tutorials.  I chose to organize them according to principle.  I think it works.

### CODE REVIEW

**Comment by chaselgrove** [comment]
Closes #124.

**Comment by asmacdo** [comment]
This needs to remove the GH 124 comment in the page as well.

File: content/resources/tutorials/_index.md [hunk 1]
```diff
@@ -8,4 +8,33 @@ TODO: [GH 124](https://github.com/ReproNim/repronim.org/issues/124)
...
+* Set up a data collection and annotation framework for a multisite study using [ReproSchema](https://repronim.netlify.app/resources/tutorials/reproschema/).
+
+**Principle 2: Data and metadata management**
+
+      * Actions: Standards, Annotation and Provenance, Version control 
+
+* *Basic:*  Using ReproNim tools to [convert data to the BIDS standard](/resources/tutorials/dicom-to-bids/) and [create a data dictionary](/resources/tutorials/data-dictionary/)
+* *Basic+:*  Adding standards and semantics to data dictionaries to promote data harmonization and findability
+* *Advanced:* Putting your derived data into BIDS derivatives and creating a semantically-enriched  data dictionary using NIDM, a standard for annotating neuroimaging data
+* *Advanced:* [DataLad containers/run \+, YODA principles](/resources/tutorials/repronim-containers/)
```
Comment (asmacdo) [inline content/resources/tutorials/_index.md:25]: Datalad containers/run + YODA, I think we don't need a comma.

Same for the next reference to the same tutorial.

**Review by asmacdo — COMMENTED** [review]
I really like this approach thanks @memartone

**Pull request merged**